openlibrary/templates/account/borrow.html

```diff
@@ -196,8 +194,9 @@ <h1>$_("Books You're Waiting For")</h1>
         </thead>
         <tbody>
         $for record in waitinglist:
-            $ book = record.get_book()
-            $ status = record['status']
+          $ book = record.get_book()
```

Comment:
fixes bug with loans page (if patron on waitlist for a sponsored book which isn't available yet)